modules/afsocket/afsocket-source.h

```diff
@@ -40,8 +41,13 @@ struct _AFSocketSourceDriver
   guint32 connections_kept_alive_across_reloads:1,
           window_size_initialized:1;
   struct iv_fd listen_fd;
+  struct iv_timer dynamic_window_timer;
+  gsize dynamic_window_timer_tick;
+  gfloat dynamic_window_stats_freq;
+  gint dynamic_window_realloc_ticks;
   gint fd;
   LogReaderOptions reader_options;
+  DynamicWindowCounter *dynamic_window_ctr;
   LogProtoServerFactory *proto_factory;
```

Comment:
I think it would make sense to move all related attributes here in the same struct for easy embedding into call-sites. I expect the very same could be used in wildcard-file() sources as well.